browser-interface/packages/lib/browser/persistentStorage.ts

```diff
@@ -1,9 +1,7 @@
-import type { PersistentAsyncStorage } from 'kernel-web-interface'
-
 declare let window: any
 
-class PersistentLocalStorage implements PersistentAsyncStorage {
-  storage: any
+class PersistentLocalStorage {
```

Comment:
Any reason that we're not using the interface? We could create a function that returns the interface and avoid using the `class`.

Basically, to avoid the following code above.
```
      setPersistentStorage(options.kernelOptions.persistentStorage as any)
```

I would like to have strong-typing